collector/lib/CollectorConfig.h

```diff
@@ -188,6 +188,8 @@ class CollectorConfig {
   void HandleAfterglowEnvVars();
   void HandleConnectionStatsEnvVars();
   void HandleSinspEnvVars();
+  void HandleConfigMapString(const std::string& jsonString);
+  void HandleConfigMap(const std::filesystem::path& filePath);
```

Comment:
Rename this so `ConfigMap` is not part of the name
```suggestion
  void HandleConfigString(const std::string& jsonString);
  void HandleConfig(const std::filesystem::path& filePath);
```